client/fidelity/compliance-review/views/compliance-review.html

```diff
@@ -10,12 +10,12 @@
             <span class="subnav__stretch-bar subnav__stretch-bar--right">
                 <div class="subnav__button-stack subnav__button-stack--padded">
                     <div class="button-list">
-                        <a
+                        <a  href
```

Comment:
Use <button> instead of <a> when you aren't creating a link.

Reply:
Done, I thought this kind of button was only made with <a> (as that's how we do it on client-core)

Reply:
I don't know where we do that in client-core, but we shouldn't do it anywhere :slightly_smiling_face: